core/lib/spree/testing_support/factory_bot.rb

```diff
@@ -44,7 +44,7 @@ def self.deprecate_cherry_picking
       def self.check_version
         require "factory_bot/version"
 
-        requirement = Gem::Requirement.new("~> 4.8")
+        requirement = Gem::Requirement.new(">= 4.8")
```

Comment:
Have we tried it still works with 4.x?

Reply:
I think this only adds explicit to handle behaviours that changed, so it should be fine.

Reply:
@waiting-for-dev I just ran the all specs against `factory_bot` version `4.8.2` and there are no failures, that's the only `4.x` version that's published from what I can see.

Reply:
Thanks, @forkata! :heart: